### ISSUE TITLE
TelegramObject Constructor Not Accepting Array or Empty Argument

### ISSUE THREAD

**Issue by ilkerkocatepe** [setiawanhu/laravel-madeline-proto#10]
Hello,
Firstly, thank you for this package.
But I got an error and couldn't find a solution.

When I used array arguments;
```
$payload = new TelegramObject([
    'peer' = '@your_username',
    'message' = 'Your Message'
]);
Messages::sendMessage($payload);

```

I'm getting this error:
```
ErrorException
Array to string conversion
```

When I used Object arguments;
```
$payload = new TelegramObject();
$payload->peer = '@your_username';
$payload->message = 'Your Message';
Messages::sendMessage($payload);
```

I'm getting:
```
ArgumentCountError
Too few arguments to function Hu\MadelineProto\TelegramObject::__construct(), 0 passed in /Users/ilkerkocatepe/Desktop/PROJELER/telegram/routes/web.php on line 27 and exactly 1 expected
```

MadelineProto::getClient() is working correctly.
Thank you.
Best regards.

_Edit: I'm using Laravel 8 and PHP 7.4_

**Comment by setiawanhu** [comment]
Hello, thanks for reaching out!

I made a little mistake when defining the TelegramObject constructor. I'll fix it right away as a new minor update.

Best regards!

**Comment by ilkerkocatepe** [comment]
Thank you for your attention.

**Comment by setiawanhu** [comment]
Fixed at #12
